client/src/pages/sessions/index.tsx

```diff
@@ -310,14 +284,9 @@ export const SessionsTable = (props: { path: string }) => {
               onSort={onSort}
             />
             <TableBody>
-              {sessions.edges
-                .filter(
-                  (edge: Edge<Session>) =>
-                    showGraded || edge.node.graderGrade === null
-                )
-                .map((row, i) => (
-                  <SessionItem key={row.node.sessionId} row={row} i={i} />
-                ))}
+              {sessions.edges.map((row, i) => (
+                <SessionItem key={row.node.sessionId} row={row} i={i} />
```

Comment:
I guess if we want to give ids to row items css-lookup in cypress, easy to do right here